### PR TITLE
Fixed #163 - Pagination bar in Emails

### LINKED ISSUE #163: Pagination bar in Emails
Hi,

We have in system more as 1000 emails, but if i want go to history, the pagination nav bar is under layer which show the emails

http://screencast.com/t/JEWzNWCzBe

I think is css issue, but i didn't find the solution to fix it  :(

Miky

### PR DESCRIPTION
**Pull request by shogunpol**
Pagination bar in Emails no hidden any more
## Description

Pagination bar in Emails no hidden any more
References issue #163 

We have in system more as 1000 emails, but if i want go to history, the pagination nav bar is under layer which show the emails

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: themes/SuiteR/js/style.js [hunk 1]
```diff
@@ -258,7 +258,7 @@ $('#backtotop').click(function(event) {
 });
 
 // Tabs jQuery for Admin panel
-$(function() {
+$(function() {j
```
Comment (mattlorimer) [inline themes/SuiteR/js/style.js]: this can't be right to add a 'j' here

File: themes/SuiteR/css/yui.css [hunk 2]
```diff
@@ -3660,46 +3656,36 @@ a.yuimenuitemlabel:hover {
...
-    {
-    color: #a6a6a6;
-}*/
-
 .yui-pg-page {
-    background-color: #fff;
-    border: 1px solid #CBCBCB;
-    padding: 2px 6px;
+    background-color: rgb(246, 246, 246);
+    padding: 0px 0px;
```
Comment (mattlorimer) [inline themes/SuiteR/css/yui.css]: px not required if its 0

**Comment by LuminisIndia** [comment]
I have upgraded to SuiteCRM 7.8.3 and came to notice of this email pagination problem with my implementation.  Can you suggest that the above mentioned commits work with this.  Reason of asking is that I have attempted to follow the changes prescribed here, but didn't find it to work.  I also attempted to make the changes suggested for themes/SuiteP/css/yui.css to 

themes/SuiteP/css/yui.scss, as I could not find the above mentioned file there.

Friends, please guide as I, like many other users, heavily prefer theme SuiteP.  Your help is solicited.

With thanks,

RK

**Comment by Dillon-Brown** [comment]
Assessed

**Comment by samus-aran** [comment]
If travis fails I would be impressed due to just having style changes....

**Pull request merged**